packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -262,11 +274,24 @@ export class Cascader extends PureComponent<CascaderProps, CascaderState> {
                 onKeyDown={this.onInputKeyDown}
                 onChange={() => {}}
                 suffix={
-                  focusCascade ? (
-                    <Icon name="angle-up" />
-                  ) : (
-                    <Icon name="angle-down" style={{ marginBottom: 0, marginLeft: '4px' }} />
-                  )
+                  <>
+                    {isClearable && activeLabel !== '' && (
+                      <IconButton
+                        name="times"
+                        aria-label="Clear selection"
+                        onClick={(e) => {
+                          e.preventDefault();
+                          e.stopPropagation();
+                          this.setState({ rcValue: [], activeLabel: '', inputValue: '' });
+                          this.props.onSelect('');
+                        }}
+                      />
+                    )}
+                    <Icon
+                      name={focusCascade ? 'angle-up' : 'angle-down'}
+                      style={{ marginBottom: 0, marginLeft: '4px' }}
```

Comment:
What about wrapping the suffix into `<Stack gap={0.5}>` to get rid of these custom styles?

Reply:
Great idea!!